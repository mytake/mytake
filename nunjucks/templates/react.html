{% extends "base.html" %}

{% block endbody %}
  <!-- Dependencies -->
  <script src="https://unpkg.com/react@15/dist/react.js"></script>
  <script src="https://unpkg.com/react-dom@15/dist/react-dom.js"></script>

  <!-- Main -->
  <script src="./bundle-0.0.3.js"></script>
{% endblock %}
