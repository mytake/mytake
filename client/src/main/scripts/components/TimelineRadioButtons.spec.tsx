import * as React from "react";
import * as renderer from "react-test-renderer";
import TimelineRadioButtons from "./TimelineRadioButtons";

const mockFn = jest.fn();

test("Debates selected", () => {
  const tree = renderer
    .create(
      <TimelineRadioButtons onChange={mockFn} selectedOption={"Debates"} />
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});

test("Documents selected", () => {
  const tree = renderer
    .create(
      <TimelineRadioButtons onChange={mockFn} selectedOption={"Documents"} />
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
