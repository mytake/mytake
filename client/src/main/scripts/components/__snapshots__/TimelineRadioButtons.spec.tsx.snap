// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Debates selected 1`] = `
<div
  className="timeline__inner-container"
>
  <div
    className="timeline__actions"
  >
    <input
      checked={true}
      className="timeline__radio"
      id="radio--debates"
      name="type"
      onChange={[MockFunction]}
      type="radio"
      value="Debates"
    />
    <label
      className="timeline__radio-label"
      htmlFor="radio--debates"
    >
      Debates
    </label>
    <input
      checked={false}
      className="timeline__radio"
      id="radio--documents"
      name="type"
      onChange={[MockFunction]}
      type="radio"
      value="Documents"
    />
    <label
      className="timeline__radio-label"
      htmlFor="radio--documents"
    >
      Founding Documents
    </label>
  </div>
</div>
`;

exports[`Documents selected 1`] = `
<div
  className="timeline__inner-container"
>
  <div
    className="timeline__actions"
  >
    <input
      checked={false}
      className="timeline__radio"
      id="radio--debates"
      name="type"
      onChange={[MockFunction]}
      type="radio"
      value="Debates"
    />
    <label
      className="timeline__radio-label"
      htmlFor="radio--debates"
    >
      Debates
    </label>
    <input
      checked={true}
      className="timeline__radio"
      id="radio--documents"
      name="type"
      onChange={[MockFunction]}
      type="radio"
      value="Documents"
    />
    <label
      className="timeline__radio-label"
      htmlFor="radio--documents"
    >
      Founding Documents
    </label>
  </div>
</div>
`;
